front/src/pods/hotel-edit/hotel-edit.validation.spec.ts

```diff
@@ -0,0 +1,238 @@
+import { FieldValidationResult } from 'lc-form-validation';
+import { HotelEditFormValidation } from './hotel-edit.validation';
+
+describe('Hotel Edit Validation tests', () => {
+    describe('Field Validations =>', () => {
+        it('should invalidate when hotel name does not exist', (done) => {
```

Comment:
Since we are checking field by field, may be it could make sense to create a nested describe set per field